proto/hummock.proto

```diff
@@ -116,6 +118,7 @@ message HummockVersion {
     uint64 group_id = 3;
     uint64 parent_group_id = 4;
     repeated uint32 member_table_ids = 5;
+    uint32 vnode_partition_count = 6;
```

Comment:
Would you explain when this field is initialized / set?

Reply:
set in function `build_initial_compaction_group_levels`.

Reply:
So for existing compaction groups, `vnode_partition_count` is 0 and cannot be changed.

Reply:
🤔 Good catch. 
I will refactor this code to avoid compatibility issues